app/templates/components/layer-record-views/zoning-for-accessibility.hbs

```diff
@@ -0,0 +1,23 @@
+<label class="header-type">
+  ZONING FOR ACCESSIBILITY
+</label>
+<h1 class="header-large uppercase">
+  Transit Station Buffer
+</h1>
+
+<p>
+  Transit-adjacent areas where applicable developments and enlargements are subject to additional consultation with the
+  Metropolitan Transportation Authority or other transit agencies with jurisdiction over the nearest mass transit station.
+</p>
+
+<p>
+  <a
+    href="https://www.nyc.gov/content/planning/pages/our-work/plans/citywide/elevate-transit-zoning-accessibility"
+    target="_blank"
+    rel="noopener noreferrer">
+    <FaIcon @icon="external-link-alt" />
+    Zoning for Accessibility&hellip; <small>(PDF)</small>
```

Comment:
Why the ellipsis and "(PDF)"?
<img width="898" alt="image" src="https://github.com/user-attachments/assets/8ea35199-5d63-4945-91dd-65100469d7bd" />

Reply:
Ok, let's remove it to stay consistent with the designs.